src/widget/translationAnimator.ts

```diff
@@ -148,7 +140,7 @@ export class TranslationAnimator extends Clutter.Actor {
         const transitionConfig: TransitionConfig = {
             duration: 250,
             mode: Clutter.AnimationMode.EASE_OUT_QUAD,
-            onStopped: () => {
+            onComplete: () => {
```

Comment:
I still think `onStopped` makes sense here. It shouldn't stop in most cases, but if it does, running `endTransition` is the correct thing to do, isn't it?

Reply:
The issue is that overriding the current transition does call `onStopped` of the previous one which mean if we transitioning a direction and then change our mind and transitioning back to the other direction it would call end transition making it jump to it's possition